static/app/views/replays/list/replaySearchAlert.tsx

```diff
@@ -0,0 +1,107 @@
+import {useMemo} from 'react';
+
+import Alert from 'sentry/components/alert';
+import ExternalLink from 'sentry/components/links/externalLink';
+import {tct} from 'sentry/locale';
+import type {Project, ProjectSdkUpdates, UpdateSdkSuggestion} from 'sentry/types';
+import {semverCompare} from 'sentry/utils/profiling/units/versions';
+import {decodeScalar} from 'sentry/utils/queryString';
+import {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import {useLocation} from 'sentry/utils/useLocation';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import useProjects from 'sentry/utils/useProjects';
+import withSdkUpdates from 'sentry/utils/withSdkUpdates';
+
+interface ReplaySearchAlertProps {
+  sdkUpdates?: ProjectSdkUpdates[] | null;
+}
+
+const MIN_REPLAY_CLICK_SDK = '7.44.0';
+
+// exported for testing
+export function ReplaySearchMinSdkAlert({sdkUpdates}: ReplaySearchAlertProps) {
+  const {selection} = usePageFilters();
+  const projects = useProjects();
+  const location = useLocation();
+  const conditions = useMemo(() => {
+    return new MutableSearch(decodeScalar(location.query.query, ''));
+  }, [location.query]);
+
+  const hasReplayClick = conditions.getFilterKeys().some(k => k.includes('click'));
+
+  if (!hasReplayClick) {
+    return null;
+  }
+
+  const selectedProjectsWithSdkUpdates = sdkUpdates?.reduce((acc, sdkUpdate) => {
+    if (!selection.projects.includes(Number(sdkUpdate.projectId))) {
+      return acc;
+    }
+
+    const project = projects.projects.find(p => p.id === sdkUpdate.projectId);
+    // should never really happen but making ts happy
+    if (!project) {
+      return acc;
+    }
+
+    acc.push({
+      project,
+      sdkUpdate,
+    });
+
+    return acc;
+  }, [] as Array<{project: Project; sdkUpdate: ProjectSdkUpdates}>);
+
+  const doesNotMeetMinSDK =
+    selectedProjectsWithSdkUpdates &&
+    selectedProjectsWithSdkUpdates.length > 0 &&
+    selectedProjectsWithSdkUpdates.every(({sdkUpdate}) => {
+      return semverCompare(sdkUpdate.sdkVersion, MIN_REPLAY_CLICK_SDK) === -1;
+    });
+
+  if (!doesNotMeetMinSDK) {
+    return null;
+  }
+
+  const sdkUpdateAction = selectedProjectsWithSdkUpdates?.[0]?.sdkUpdate.suggestions.find(
+    suggestion => suggestion.type === 'updateSdk'
+  ) as UpdateSdkSuggestion | undefined;
+
+  if (sdkUpdateAction) {
+    return (
+      <Alert data-test-id="min-sdk-alert">
```

Comment:
`data-test-id` doesn't seem to be used in the test. 

i don't mind if it stays around tbh, but it's been pointed out to me enough when i leave it, so i'm paying it forward.

Reply:
i've added an assertion for this